### ISSUE TITLE
Chest X-ray - 17 Diseases Prediction

### ISSUE THREAD

**Issue by kyra-09** [TAHIR0110/ThereForYou#138]
**Is your feature request related to a problem? Please describe.**
The project is valuable for research in neurology, radiology, and oncology. It allows the development and evaluation of computer-based algorithms, machine learning models, and deep learning techniques for automated detection, diagnosis, and classification of the condiitons for diseases related to chest X-rays.

**Describe the solution you'd like**
   1. Data Preparation: Collect and preprocess a balanced dataset of real and fake face images, including normalization, resizing, and augmentation.
  2.  Base Model Selection:  EfficientNetB0,VGG16 , Xception , InceptionV3 like 5 different models  excluding its top layers, to leverage its learned features.
  3.  Model Construction: Add custom layers on top of the base model for binary classification, compiling with appropriate loss and metrics.
  4.  Initial Training: Train the model with the base layers frozen to only update the new layers.
  5.  Fine-Tuning: Unfreeze some or all of the base model layers and continue training with a lower learning rate to fine-tune the entire network.
  6.) EDA analysis.
  7.) Comaprioson using performance matrices such as accuracy scores , confusion matrix etc.
  
Dataset I'll use :- https://www.kaggle.com/datasets/trainingdatapro/chest-xray-17-diseases

**Describe alternatives you've considered**

**Additional context**

**Comment by kyra-09** [comment]
@TAHIR0110 kindly assign this issue to me.

**Comment by TAHIR0110** [comment]
@kyra-09 assigned as level3!
